symex-interface-fennel.el

```diff
@@ -29,7 +29,7 @@
 (require 'symex-interop)
 
 
-(declare-function fennel-find-definition  "ext:fennel-mode")
+[(declare-function fennel-find-definition  "ext:fennel-mode")]
```

Comment:
Is this change to add surrounding square brackets intentional? I'm not familiar with this syntax, what does it do?

Reply:
Ho I am sorry, this is totally unintentional, I should have double checked the diff, I will remove this !